lib/rucio/db/sqla/models.py

```diff
@@ -353,6 +353,7 @@ class DataIdentifier(BASE, ModelBase):
     eol_at = Column(DateTime)
     is_archive = Column(Boolean(name='DIDS_ARCHIVE_CHK'))
     constituent = Column(Boolean(name='DIDS_CONSTITUENT_CHK'))
+    nbaccesses = Column(Integer())
```

Comment:
just a minor comment, i've rarely seen other projects use "nbsomething", it always seems to be "nr_something".

but i have no particular strong opinion on this

Reply:
Actually, to be most consistent with other names in our schema, access_cnt is probably the best name, as we have several other columns named _cnt

Reply:
👍